WIP started handbook v2 content

jamesefhawkins opened the pull request:
## Changes

Creating a more concise overview. plan is to treat handbook more like technical docs - interesting tutorial/walk through referencing appendix material. A big job.


## Checklist
- [ ] Titles are in [sentence case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case)
- [ ] Feature names are in **[sentence case too]([https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case))**
- [ ] Words are spelled using American English
- [ ] I have checked out our [style guide](https://github.com/PostHog/posthog.com/blob/master/STYLEGUIDE.md)
- [ ] If I moved a page, I added a redirect in `vercel.json`

jamesefhawkins commented on contents/handbook/getting-started/world-class-engineering.md:
```diff
@@ -0,0 +1,41 @@
+
+One of the things we've learned is the very strongest engineers are usually those who want autonomy the most, and so freedom is a great way to attract and retain world class talent. Now that we're lucky enough to have people like this already here, people see PostHog as a destination company, accelerating further our access to some of the best people in the world at what they do.
+
+## A high percentage of the company are engineers
```
```suggestion
## A high percentage of the company are engineers
```

charlescook-ph commented:
Do you want feedback on this yet? 

(I'm planning to start working on the [Hedgehog OS stuff](https://github.com/PostHog/company-internal/issues/1110) after this is all merged to avoid making a PR mess)

corywatilo commented:
FYI there are a few references to Luke in the Handbook. Might be easier to change in this PR vs trying to reconcile a bunch of changes in these pages later.

`in-app-prompts.md`

> If you have any questions message Luke

`side-gigs.md`

> We ask you to please just check in with the relevant Exec team member for your team (ie. James, Tim, Charles, or Luke) to get their confirmation before going ahead with any side gigs. 

`small-teams.md`

> Luke is ultimately responsible for us having (i) no gaps in product (ii) eliminating duplicate work (iii) making sure all Small Teams are working on something rational. This is how we manage the product.

> Luke has the ultimate responsibility to make sure we don't build the same thing in two different teams, or that we don't accidentally compete with each other internally.

corywatilo commented:
Proposed handbook UI changes:

- Split into chapters and resources (split by department)
- Wider side nav (to support long chapter names)
- This basically assumes all content from _Getting started, Company, Strategy,_ and _How we work_ are covered in chapters. We should move each small team page to be the root of the department pages, as a landing page for each small team where you can then read more about logistics and policies and such.

<img width="1276" alt="image" src="https://github.com/PostHog/posthog.com/assets/154479/a0e60cb1-e006-4d13-84e3-d891490bcd83">

charlescook-ph commented:
Not sure if/where it fits @jamesefhawkins but I think there's something missing around what it's like to work here, centred around:

- Why writing is central to our culture
- Creating space for deep work
- Anti-meeting by default

All the 'how we communicate' stuff around PRs > Issues etc. obviously goes in the appendices, but some of this stuff feels like something we want everyone to definitely read?

corywatilo commented:
> some of this stuff feels like something we want everyone to definitely read?

I've often thought it would be handy to have a couple new sections:

- "It's your first week. Here's your quick reference guide."
  - no yolo merging
  - install the GitHub app for Slack to stay on top of things (with instructions on how to use)
  - avoid assigning issues to people
  - how to emoji like a posthogger: top custom emojis we use and their origin stories
  - suggested slack channels to join
  - inside jokes you might hear at offsites/useful knowledge (three fingers, sticky notes and how to peel them)
- "Read these monthly"
  - PRs > issues > mentioning someone in Slack
  - keep an eye on community questions and be helpful where you can
  - public channels > DMs and private groups for most things

These don't fit squarely into a) a top-to-bottom book, or b) department-related info. A "quick reference" section (covering above, and more) might be cool because they really show how we work differently.

jamesefhawkins commented:
Added a bunch as per below as i agreed with the feedback here. Not quite sure the content is in the right place, but the words are there now...

* "It's your first week. Here's your quick reference guide."
  * added to how you can help page
* no yolo merging
  * added to how you can help page
* install the GitHub app for Slack to stay on top of things (with instructions on how to use)
  * I don't use this, felt a bit too specific for the "book" but owuld be good as a separate PR somewhere elsee
* avoid assigning issues to people
   * added to how you can help page
* how to emoji like a posthogger: top custom emojis we use and their origin stories
  * added to how you can help page
* suggested slack channels to join
  * I just automated this by updating what we add people to in slack by default
* inside jokes you might hear at offsites/useful knowledge (three fingers, sticky notes and how to peel them)
  * lore of posthog section added
* PRs > issues > mentioning someone in Slack
  * added to how you can help page
* keep an eye on community questions and be helpful where you can
  * added to how you can help page
* public channels > DMs and private groups for most things
   * added to how you can help page



todos:
* I still need to add the "where we are today" page
* remove duplicate content from pre-existing pages and update internal links where relevant

corywatilo commented:
## Todos
- [ ] Fix _Next article_ block (chapter 1 links to chapter 1)
- [ ] Introduce _who_ wrote the handbook somewhere
    > I happened to go to an excellent talk by Jeff Lawson (Twilio CEO). It made me realize I should be asking - who ships more per person, a startup or an enterprise?
- [ ] How we make users happy: Missing a lead-in tying the title to the two sections, just drops right into product without context
- [ ] Not running out of money: Last sentence confuses me:
    > And that it's harder to predict how fast we'll grow.
- [ ] Future: This sentence is sort of confusing. We need 9% but we're at 7%, but we over-performed? Has the scale changed over time?
    > When we set the goal, we needed 9% monthly grow. We've overperformed, and that means as of today, it's around 7%

timgl commented on contents/handbook/making-users-happy.md:
```diff
@@ -0,0 +1,27 @@
+
+## How we build a product
+
+First, someone internally will suggest an idea. Usually this will come from James and Tim, but it has just as frequently come from anyone else on the team.
```
```suggestion
First, someone internally will suggest an idea. Sometimes this will come from James and Tim, but it has just as frequently come from anyone else on the team.
```

timgl commented on contents/handbook/making-users-happy.md:
```diff
@@ -0,0 +1,27 @@
+
+That person builds the MVP, and the only goal is to figure out if anyone will use it. With some products, the MVP may have more scope if we feel especially confident. Once the new product is in a non-embarassing state (that won't harm our brand), we add pricing to it and position it on our website. This drives much more demand. At this stage, the goal is to get the product to product-market fit i ...
+
+Once all this is done – which we'd expect to take a few months – we can start to innovate. This usually means some kind of platform play - for example, extending the product to enhance everything else we're working on, or shipping a new product that'd work well with it.
```
```suggestion
Once all this is done – which we'd expect to take a few months – we can start to innovate. This usually means some kind of platform play - for example, extending the product to enhance everything else we're working on, or shipping another new product that'd work well with it.
```

timgl commented on contents/handbook/making-users-happy.md:
```diff
@@ -0,0 +1,27 @@
+
+## Engineers talk to users and provide support
+
+If you are working on a product at PostHog that hasn't hit product-market fit, then you should be _as close as possible_ to  your users, feeling whatever they feel so you have as much signal as possible to make the product great.
+
+_Only once_ this has happened do we have centralized support. However, even once you're at this stage, this support is always from a developer so we don't needlessly triage problems from one place to another before they wind up in an engineer's hands anyway. We'd rather fewer people with deeper technical knowledge who can fix bugs and update docs as they go, since in the long run that will lead t ...
+
+No matter what stage a product, our engineers work hand in hand with our customers (also engineers) whenever they are building something.
```
This feels overly complex.  Maybe just
```suggestion
You should be _as close as possible_ to  your users, feeling whatever they feel so you have as much signal as possible to make the product great.

For established products with a lot of usage questions (how do I create an insight that does X for example), Customer Success helps with support.
```

timgl commented on src/pages/handbook.tsx:
```diff
@@ -5,6 +5,32 @@ import { StaticImage } from 'gatsby-plugin-image'
+    {
+        name: 'Chapters',
+        links: [
+            { order: '1', name: 'What is PostHog?', to: '/handbook/what-is-posthog' },
+            { order: '2', name: 'A wide company with small teams', to: '/handbook/wide-company' },
+            {
+                order: '3',
+                name: 'Providing a world-class engineering environment',
+                to: '/handbook/world-class-engineering',
+            },
+            { order: '4', name: 'Deciding which products to build', to: '/handbook/which-products' },
+            { order: '5', name: 'How we get users', to: '/handbook/how-we-get-users' },
+            { order: '6', name: 'Enduringly low prices', to: '/handbook/low-prices' },
+            { order: '7', name: 'How we make users happy', to: '/handbook/making-users-happy' },
+            { order: '8', name: 'How we make money', to: '/handbook/how-we-make-money' },
+            { order: '9', name: "How we're building a world-class team", to: '/handbook/strong-team' },
+            { order: '10', name: 'Not running out of money', to: '/handbook/finance' },
+            { order: '11', name: 'How we got here', to: '/handbook/story' },
+            { order: '12', name: 'Where we are now', to: '/handbook/current-status' },
+            { order: '13', name: 'Where are we going?', to: '/handbook/future' },
+            { order: '14', name: 'How you can help', to: '/handbook/help' },
```
I know we came up with this ordering, but it feels like jumping around here. Quick suggestion

```suggestion
            { order: '1', name: 'What is PostHog?', to: '/handbook/what-is-posthog' },
            { order: '2', name: 'How we got here', to: '/handbook/story' },
            { order: '3', name: 'How we get users', to: '/handbook/how-we-get-users' },
            { order: '4', name: 'How we make users happy', to: '/handbook/making-users-happy' },
            { order: '5', name: 'How we make money', to: '/handbook/how-we-make-money' },
            { order: '6', name: 'Enduringly low prices', to: '/handbook/low-prices' },
            { order: '7', name: 'Deciding which products to build', to: '/handbook/which-products' },
            { order: '8', name: 'A wide company with small teams', to: '/handbook/wide-company' },
            { order: '9', name: "How we're building a world-class team", to: '/handbook/strong-team' },
            {
                order: '10',
                name: 'Providing a world-class engineering environment',
                to: '/handbook/world-class-engineering',
            },
            { order: '11', name: 'Not running out of money', to: '/handbook/finance' },
            { order: '12', name: 'Where we are now', to: '/handbook/current-status' },
            { order: '13', name: 'Where are we going?', to: '/handbook/future' },
            { order: '14', name: 'How you can help', to: '/handbook/help' },
```

timgl commented on contents/handbook/help.mdx:
```diff
@@ -0,0 +1,122 @@
+
+You can find these in [posthog.com/questions](https://posthog.com/questions).
+
+## Longer term - values
```
I think this deserves a separate chapter

jamesefhawkins replied:
done

timgl commented on contents/handbook/help.mdx:
```diff
@@ -0,0 +1,122 @@
+
+Everything else is a means to this end! We often do onboarding in person to accelerate all of the above. This usually takes around a week.
+
+## General etiquette
```
Not sure these are interesting enough for this... maybe reference material?

timgl commented on contents/handbook/help.mdx:
```diff
@@ -0,0 +1,122 @@
+showTitle: true
+---
+
+## Getting yourself up and running - your first week
```
Again, probably reference material. Also highlights that it's a little bit confusing who this guide is written for. a new employee? Existing employees?

timgl commented on contents/handbook/help.mdx:
```diff
@@ -0,0 +1,122 @@
+
+We are a team, not a family. This means we have very ambitious goals, [compensate](https://posthog.com/handbook/people/compensation#how-it-works) generously and transparently, offer [exceptional benefits](/handbook/people/benefits), and do everything we can to provide an environment for you to do your best work. Often this means everyone, [especially managers](/handbook/company/management), getti ...
+
+## Lore of PostHog / inside jokes
```
Love this

timgl commented on contents/handbook/strong-team.md:
```diff
@@ -0,0 +1,40 @@
+
+When we raised our Series A, one of the first things we did was to make sure we didn't lose our existing team (at least for pay reasons!) _before_ we added more people to it. Today this is still true - we proactively review everyone's pay 3-4 times a year and increase it if people have leveled up.
+
+When it comes to churn due to pay, fairness is often more important than the absolute level. We do this in line with a transparent pay system that we even [make public](/handbook/people/compensation).
```
This first sentence makes it sound like we don't pay well.

jamesefhawkins replied:
updated

timgl commented on contents/handbook/strong-team.md:
```diff
@@ -0,0 +1,40 @@
+
+When it comes to churn due to pay, fairness is often more important than the absolute level. We do this in line with a transparent pay system that we even [make public](/handbook/people/compensation).
+
+For options, we generally offer smaller amounts, but [the most generous terms possible](/handbook/people/share-options) as it feels like the right thing to do. We think this makes it as likely as possible people can see huge upside if we are successful (making it easier to raise _and_ more realistic that people will actually get money from them). That motivates everybody.
```
Same

jamesefhawkins replied:
adjusted

timgl commented on contents/handbook/future.md:
```diff
@@ -0,0 +1,35 @@
+
+We're *not* excited by:
+
+* building a $1 billion company - we think going _much_ further is where the real challenge lies
```
Very subtle but the phrasing of this could mean you want to build a company that is smaller than $1bn
```suggestion
* selling the company for $1 billion - we think we can build a much bigger company by staying independent
```

timgl commented on contents/handbook/help.mdx:
```diff
@@ -0,0 +1,122 @@
+* <Emoji name="eu-thumbsup" src="/images/emojis/eu-thumbsup.png" /> / <Emoji name="thumbs-down-eu" src="/images/emojis/thumbs-down-eu.png" /> We once made <a href="https://www.isgoogleanalyticsillegal.com">isgoogleanalyticsillegal.com</a> when there were privacy rulings about Google Analytics. We put it on HN, got the top of the front page and it was our biggest <em>ever</em> day of signups at th ...
+* IPO promises. There is a list of these that is brought out at certain moments. You may see.
+* [Marius](/community/profiles/1) will train you on Post-it notes if you go to an offsite with him. Success of a good Post-it note posting is in the lift away from the surface - the most important thing is to peel, as opposed to pulling, the Post-it note off.
+* Three finger rule - another Marius invention, if someone holds up three fingers while you're talking, it means you aren't being concise enough. We don't actually use this much as it's predictably awkward and distracting, so ruins any meeting it could have otherwise helped.
```
I think the "How you can help" page should end with "we're hiring, come join us!" as a sign off

jamesefhawkins replied:
done

timgl reviewed (commented):
This is very good. A lot of smaller thoughts but the majority is there
